src/api/java/com/minecolonies/api/entity/citizen/AbstractEntityCitizen.java

```diff
@@ -734,4 +740,24 @@ public void updatePose(final Pose pose)
     {
         setPose(pose);
     }
+
+    /**
+     * Disallow pushing from fluids to prevent stuck
+     *
+     * @return
+     */
+    public boolean isPushedByFluid()
+    {
+        return false;
+    }
+
+    /**
+     * Do not allow bubble movement
+     *
+     * @param down
+     */
+    public void onInsideBubbleColumn(boolean down)
```

Comment:
Maybe we want to allow upward movement and not downward?

Reply:
they do swim upwards by themselves, they just no longer will be hindered in where they want to move by the column itself